letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -410,8 +410,7 @@ BootstrapRpmCommon() {
     ca-certificates
   "
 
-  # Some distros and older versions of current distros use a "python27"
-  # instead of "python" naming convention. Try both conventions.
+  # Most RPM distros use the "python naming convention".  Let's try that first.
```

Comment:
nit: rather than `"python naming convention"`, it might be more clear to use `"python" naming convention` or `"python-" naming convention`.